src/App.css

```diff
@@ -10,6 +10,26 @@
   letter-spacing: 1px;
 }
 
+.header-settings-btn {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 2px;
+  border: none;
+  background: transparent;
+  cursor: pointer;
+}
+
+.header-settings-btn {
+  right: 20px;
```

Comment:
Adding `top: 6px;` to this ruleset fixes it when I apply that locally. Might not be an ideal fix, but maybe good enough?

Also, I'm happy to go ahead and approve this and we can fix the button placement later!